README.md

```diff
@@ -126,8 +130,6 @@ is correct, while
 
 is not, because the `-d` argument applies to the `cd` command.
 
-~~Check the man page (`whipper(1)`) for more information.~~ (currently not available as whipper's documentation is planned to be reworked ([Issue #73](https://github.com/JoeLametta/whipper/issues/73))).
```

Comment:
We can add mention of a man page when we have one, but I don't think there's any need to call attention to the fact that it's a planned feature.